assets/css/hazard-detail-card.css

```diff
@@ -0,0 +1,51 @@
+#hazard-card__container {
+  position: absolute;
+  z-index: 9;
+}
+
+#hazard-card__outer {
+  position: absolute;
+}
+
+.report-card__top-controls {
+  display: flex;
+  width: 21.4375rem;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 1rem;
+}
+
+.circle-border {
+  display: flex;
+  width: 2rem;
+  height: 2rem;
+  padding: 0.4rem;
+  justify-content: center;
+  align-items: center;
+  flex-shrink: 0;
+  border-radius: 6.25rem;
+  border: 1px solid var(--neutral-neutral-300, #d0d5dd);
+  background: var(--white, #fff);
+}
+
+.report-card__hazard-detail-buttons {
+  display: flex;
+  align-items: flex-start;
+  align-content: flex-start;
+  gap: 0.75rem;
+  align-self: stretch;
+  flex-wrap: wrap;
+}
+
+.report-card__user-details {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  align-self: stretch;
+}
+
+#user-image {
+  width: 1.75rem;
+  height: 1.75rem;
+  border-radius: 6.25rem;
+}
```

Comment:
#report-card__picture-container {
  overflow: hidden;
}

Reply:
Thanks! I used `overflow: scroll;` to match the designs, but also added some width constrains to make it function correctly.